app/main.py

```diff
@@ -2,0 +1,12 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, source, destination = command.split()
+    if command != "mv":
+        return
+    path_, new_file_name = os.path.split(destination)
+    if path_ and not os.path.exists(path_):
+        os.makedirs(path_)
+    shutil.move(source, destination)
```

Comment:
Clean Code: It's a good practice to add comments to your code. This will help other developers understand your code better.